manim/mobject/types/image_mobject.py

```diff
@@ -27,8 +27,8 @@ class AbstractImageMobject(Mobject):
     ----------
     scale_to_resolution : :class:`int`
         At this resolution the image is placed pixel by pixel onto the screen, so it will look the sharpest and best.
-        This is a custom parameter of ImageMobject so that rendering a scene with the `--quality low` flag for faster rendering and testing won't effect the position of the image on the screen.
-        Calculated by `height = image_height / scale_to_resolution * config["frame_height"]`
+        This is a custom parameter of ImageMobject so that rendering a scene with the ``--quality low`` flag for faster rendering and testing won't effect the position of the image on the screen.
```

Comment:
Nitpick: This line reads as if `scale_to_resolution` only plays a role when rendering with `--quality low`. IMHO, it could be deleted (which is what I suggested in the previous PR).

Reply:
Ok, I change this line to a more general formulation then.
However, I would not delete it, because I think it gives additional clarity over what is happening.